### DISCUSSION TITLE
how to import the `new` proc macro? following the examples does not work

### DISCUSSION THREAD

**Discussion by vincenzopalazzo** [PyO3/pyo3#4700] [Questions] [answered]
```
error: cannot find attribute `new` in this scope
  --> fundamentals/src/bolt/bolt1.rs:31:34
   |
31 |     #[cfg_attr(feature = "pyo3", new)]
   |                                  ^^^
```

looks like that it is not available in `0.20` and `0.22` tried out also the `use pyo3::prelude::*;` but no change.

I have a single package with just `maturin new -b pyo3 foo`

**Comment by messense** [comment]
See https://github.com/PyO3/pyo3/issues/780 and the discussion in https://github.com/PyO3/pyo3/pull/2786

**Comment by vincenzopalazzo** [comment]
@messense what I am missing of the pointer that you sent to me there is a reason why `new` in not exported in `py03`? My example does not work also without `cfg_attr`

`use py03::new` say that there is no `new`

**Comment by messense** [comment, marked as answer]
`new` is merely an annotation of [`pymethods`](https://docs.rs/pyo3/latest/pyo3/attr.pymethods.html), there is nothing to import.

**Reply by vincenzopalazzo** [reply]
Ah reading the docs in the wrong way. Thanks!
